pcsx2/GS/GS.rc

```diff
@@ -240,7 +240,7 @@ END
 
 IDD_CONFIG DIALOGEX 0, 0, 242, 315
 STYLE DS_SETFONT | DS_MODALFRAME | DS_FIXEDSYS | WS_POPUP | WS_CAPTION | WS_SYSMENU
-CAPTION "GS Settings"
+CAPTION "Graphical Settings"
```

Comment:
I have an open PR where the main PCSX2 window will refer to this as `Graphics Settings` instead.